.github/workflows/sync-pangeobot-fork.yml

```diff
@@ -0,0 +1,16 @@
+on:
+  schedule:
+  - cron:  "*/5 * * * *"
```

Comment:
rather than a cron job, can we just have this run on each commit to master?

Reply:
that's what i originally planned but unfortunately i don't think we can... if the workflow were to run in *this* repository (commits to master) I think we need the ability to add secrets to dispatch to a workflow in the pangeo-bot fork (https://github.com/marketplace/actions/git-sync-action or https://github.com/peter-evans/repository-dispatch.